cgc/cicero.cwl

```diff
@@ -0,0 +1,156 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  sbg: 'https://sevenbridges.com'
+id: an.thrasher/cicero-benchmark/cicero/8
```

Comment:
Can you remove the username + project information? We should probably just use `id: cicero`.

```suggestion
id: cicero
```

Reply:
I probably wouldnt until we have the permanent namespace finished...this was all generated by CGC